src/__tests__/__apps__/basic/package.json

```
@@ -1,18 +1,18 @@
 {
-  "name": "local-module",
+  "name": "test-local-module",
   "version": "1.0.0",
   "description": "",
-  "main": "index.js",
+  "private": "true",
   "scripts": {
-    "dev": "next",
-    "build": "next build",
-    "start": "next start"
+    "dev": "cross-env NODE_PRESERVE_SYMLINKS=1 next",
+    "build": "cross-env NODE_PRESERVE_SYMLINKS=1 next build",
+    "start": "cross-env NODE_PRESERVE_SYMLINKS=1 next start"
   },
-  "author": "",
-  "license": "ISC",
+  "author": "Pierre de la Martinière <pierre.de.la.martiniere@gmail.com>",
+  "license": "MIT",
   "dependencies": {
     "lodash-es": "^4.17.15",
-    "next": "9.3.0",
+    "next": "9.4.5-canary.45",
```

```suggestion
    "next": "9.5.0",
```
This should probably be 9.5 :)

---

yes, it was not released at the time